internal/hud/renderer.go

```diff
@@ -220,6 +220,9 @@ func renderResource(r view.Resource) rty.Component {
 		l.Add(rty.ColoredString("    K8S: ", cLightText))
 		l.Add(rty.TextString(fmt.Sprintf("Pod [%s] • %s ago — ", r.PodName, formatDuration(time.Since(r.PodCreationTime)))))
 		l.Add(rty.ColoredString(r.PodStatus, podStatusColor))
+		if r.PodRestarts > 0 {
```

Comment:
it might be good to also show # restarts even when it's 0, but make it gray or green instead of red

this might be bikesheddy and don't bother if you don't feel compelled